### DISCUSSION TITLE
Any way to limit notifications on certain clients or agents to certain times?

### DISCUSSION THREAD

**Discussion by joeystombaugh** [amidaware/tacticalrmm#1453] [Ideas]
Say I have an office setting that has multiple computers that are on all night, but overnight they might have high memory usage because of updates or programs being left on etc.. There should be a way to only get alerts about problems during a certain time-frame for clients or agents.

**Comment by wh1te909** [comment]
not atm but it is planned, see https://github.com/amidaware/tacticalrmm/issues/1356 and https://github.com/amidaware/tacticalrmm/issues/1064
